site/en/docs/android/custom-tabs/index.md

```diff
@@ -3,93 +3,117 @@ layout: "layouts/doc-post.njk"
-
-## When should I use Custom Tabs vs WebView?
-
-The WebView is good solution if you are hosting your own content inside your
-app. If your app directs people to URLs outside your domain, we recommend
-that you use Custom Tabs for these reasons:
-
-- Support for the same web platform features and capabilities as the browsers.
-- Simple to implement. No need to build code to manage requests, permission
-	grants or cookie stores.
-- UI customization:
-	- Initial launch height
-  	- Toolbar color
-	- Action button
-	- Custom menu items
-	- Custom in/out animations
-	- Bottom toolbar
-	- Close button position
-	- Corner radii
-- Navigation awareness: the browser delivers a callback to the application upon
-an external navigation.
-- Security: the browser uses Google's Safe Browsing to protect the user and the device from
-dangerous sites.
-- Performance optimization:
-	- Pre-warming of the Browser in the background, while avoiding stealing
+Custom Tabs are a feature in modern Android browsers that gives app developers
+a way to add a customized browser experience directly within their app.
+
+Loading web content has been a part of mobile apps since the early days of
+smartphones, but older options can present challenges for devs. Launching the
+actual browser is a heavy context switch for users that isn't customizable,
+while WebViews [don't support][6] all features of the web platform, don't share
+state with the browser and add maintenance overhead.
+
+Custom Tabs offer a better user experience than simply opening an external
+browser. They allow users to remain within the app while browsing, increasing
+engagement and reducing the risk of users abandoning the app. They accomplish
+this by by being powered directly by the user's preferred browser, automatically
+sharing the state and features offered by it. You don't need to write custom
+code to manage requests, permission grants or cookie stores.
+
+
+## What can Custom Tabs do?
+
+By using a Custom Tab, your web content will load in whatever rendering engine
+powers your user's preferred browser. Any API or web platform feature is
+available there, will be available in your Custom Tab. Their browsing session,
+saved passwords, payment methods, and addresses will all show up just like they
+are accustomed already.
+
+## What can I customize in a Custom Tab?
+
+Quite a bit! Custom Tabs give you fine grained control over a lot of the browser
+chrome and user experience. Within your app, you launch a Custom Tab via an [Intent](7).
+When this Intent is called, you can add a number of attributes to the
+[CustomTabIntent](8) to get the exact experience you want. This includes
+
+Custom Entrance and Exit animations to match the rest of your app
+
+<figure class="screenshot">
+{% Video preload=true, loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/sIeKPXwrHXdCXtGRrv2Q.mp4", width="350", height="730", caption="" %}
```

Comment:
For all img/video: `class="screenshot screenshot--filled"` should go on the video / img shortcode, not the figure.